monitoring/src/main/scala/akkaviz/serialization/ClassInspector.scala

```diff
@@ -56,23 +60,33 @@ object ClassInspector {
     new ClassInspector {
       val underlyingClass: Class[_] = clazz
       private[this] val f = reflectedFields.toSeq
+      private[this] val accessors: List[Accessor] = f.flatMap {
+        cf =>
+          try {
+            val field = underlyingClass.getDeclaredField(cf.name)
+            field.setAccessible(true)
+            Some(Accessor(cf.name, field))
+          } catch {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Fatal exception is being caught](https://www.codacy.com/app/lustefaniak/akka-viz/file/2215348010/issues/source?bid=3121282&fileBranchId=3172196#l70)